src/entity/User.ts

```diff
@@ -51,9 +51,6 @@ export class User {
   @Column({ length: 39, nullable: true })
   hashnode?: string;
 
-  @Column({ default: false })
-  devcardEligible: boolean;
```

Comment:
I wouldn't recommend actually dropping this column. I'm not a fan of deleting data 😓

Reply:
No worries, let me put the column back and remove the migration for now :)